cli-commands/compiler/etherlime-contract-sources/index.js

```diff
@@ -1,26 +1,51 @@
 const dir = require("node-dir");
 const path = require("path");
 const fs = require("fs");
+const VYPER_EXTENSION = require('../vyper-compiler/config.js').VYPER_EXTENSION;
+const SOL_EXTENSION = '.sol';
```

Comment:
I'd rather move this constant in a config file that is not only for the vyper compiler, but the compiler in general